mms/model_service_worker.py

```diff
@@ -442,6 +442,9 @@ def run_server(self):
         :return:
         """
         try:
+            self.sock.settimeout(SOCKET_ACCEPT_TIMEOUT)
+            self.sock.setblocking(True)  # workaround error(35, 'Resource temporarily unavailable') on OSX
```

Comment:
Is it by set to non-blocking after setting a timeout?

Reply:
Some how it messed up socket opt in some version of python on OSX.